packages/babel-parser/test/fixtures/flow/type-annotations/20/output.json

```diff
@@ -34,7 +34,6 @@
                 },
                 "computed": false,
                 "kind": "set",
-                "variance": null,
```

Comment:
The `variance` property exists in ClassProperty, and we don't have `variance` for ObjectMethod. This PR removes `variance` of `fooProp` accessor for the following cases:
```js
a={set fooProp(value:number){}}
```

I think the new behaviour is preferred and it aligns to what we did for non-accessors.